Arena.toml

```diff
@@ -1667,27 +1667,37 @@ permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06
 
 [[diagnostics]]
 document = "stjudecloud/workflows:/tools/kraken2.wdl"
-message = "kraken2.wdl:120:20: note[ContainerValue]: container URI uses a mutable tag"
+message = "kraken2.wdl:120:20: note[ContainerUri]: container URI uses a mutable tag"
 permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/kraken2.wdl/#L120"
 
 [[diagnostics]]
 document = "stjudecloud/workflows:/tools/kraken2.wdl"
-message = "kraken2.wdl:177:20: note[ContainerValue]: container URI uses a mutable tag"
+message = "kraken2.wdl:177:20: note[ContainerUri]: container URI uses a mutable tag"
 permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/kraken2.wdl/#L177"
 
 [[diagnostics]]
 document = "stjudecloud/workflows:/tools/kraken2.wdl"
-message = "kraken2.wdl:287:20: note[ContainerValue]: container URI uses a mutable tag"
+message = "kraken2.wdl:287:20: note[ContainerUri]: container URI uses a mutable tag"
 permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/kraken2.wdl/#L287"
 
 [[diagnostics]]
 document = "stjudecloud/workflows:/tools/kraken2.wdl"
-message = "kraken2.wdl:404:20: note[ContainerValue]: container URI uses a mutable tag"
+message = "kraken2.wdl:333:20: note[KnownRules]: unknown lint rule `DisallowedInputName`"
+permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/kraken2.wdl/#L333"
```

Comment:
@a-frantz - We'll have to make these changes in `workflows` as well once this is merged.

Reply:
not till the next sprocket release. Which means we'll have some extra entries in Arena until the next release, but oh well 🤷‍♀️